packages/calypso-e2e/src/lib/components/editor-list-view-component.ts

```diff
@@ -0,0 +1,35 @@
+import { Locator, Page } from 'playwright';
+
+const selectors = {
+	blockLink: ( blockName: string ) => `.block-editor-list-view-leaf a:has-text("${ blockName }")`,
+};
+
+/**
+ * Represents an instance of the WordPress.com Editor's sidebar list view.
+ * The component is available only in the Desktop viewport.
+ */
+export class EditorListViewComponent {
```

Comment:
Is this a further subcomponent-ization of the navsidebar or am I misunderstanding it?

Reply:
It's just a completely different sidebar! The nav sidebar is the dark-colored one that is for navigating back to calypso.

This is the sidebar view that shows all the blocks on your post arranged in list view, to quickly jump between them and re-arrange them!

Reply:
> The nav sidebar is the dark-colored one that is for navigating back to calypso.

At least, that's how I've always understood it.

Reply:
Interesting, didn't know that was a thing!

Reply:
Given this is a list view of blocks in a post/page, how about a name that incorporates the term "block" somehow?